hal_st/stm32fxxx/WatchDogStm.hpp

```diff
@@ -11,12 +11,25 @@ namespace hal
     class WatchDogStm
     {
     public:
-        WatchDogStm(const infra::Function<void()>& onExpired);
+        enum Prescalers
```

Comment:
I am not sure this enum is required. Usually the approach is a `uint32_t` and you'd set it to `WWDG_PRESCALER_xx` in your Main.cpp